script.js

```diff
@@ -1,12 +1,78 @@
 //You can edit ALL of the code here
+let allEpisodes;
 function setup() {
-  const allEpisodes = getAllEpisodes();
+  allEpisodes = getAllEpisodes();
   makePageForEpisodes(allEpisodes);
 }
 
 function makePageForEpisodes(episodeList) {
   const rootElem = document.getElementById("root");
-  rootElem.textContent = `Got ${episodeList.length} episode(s)`;
+  rootElem.textContent = ""
+  const episodeCount= document.getElementById("episodeCount");
+  episodeCount.innerHTML= `Displaying ${episodeList.length}/73 episodes`;
+  //create select input
+  const selectEl=document.getElementById("episodeOptions");
+
+  episodeList.forEach(episode=>{
+    
+    //create body element
+    const divEl=document.createElement("div");
+    divEl.classList.add("container")
```

Comment:
```suggestion
    divEl.className ="container";
```
Both are correct the only difference is className is used to add just one class (which is what you need in your case) and classList to add multiple classes